types/file-saver/index.d.ts

```diff
@@ -4,6 +4,7 @@
 //                 Daniel Roth <https://github.com/DaIgeb>
 //                 Chris Barr <https://github.com/chrismbarr>
 //                 HitkoDev <https://github.com/HitkoDev>
+//                 JounQin <https://github.com/JounQin>
```

Comment:
Just to confirm: this list isn't for crediting contributors - it's a list of people who want to be notified when this package changes so that they can provide reviews.

Reply:
Yes, I understand.